tools/putbin/putbin.c

```diff
@@ -304,10 +305,10 @@ main(int argc, char **argv)
                 vlen = (unsigned)strtoul(optarg, 0, 0);
```

Comment:
Seems like you can get rid of the same cast here?

Reply:
yes, will do.

Reply:
oh wait, _vlen_ is an unsigned (int), so it will need the cast. I got rid of the cast for variables that I changed to unsigned longs.

Reply:
You shouldn't need the cast unlesss you have the warning enabled to warn of lossy assignments, otherwise C will perform an implicit cast..

Reply:
Which flag enables that?  I looked (briefly) and didn't see it.